client/lib/upgrades/actions/purchases.js

```diff
@@ -9,66 +9,21 @@ import i18n from 'i18n-calypso';
  */
 import { action as ActionTypes } from '../constants';
 import Dispatcher from 'dispatcher';
-import olark from 'lib/olark';
-import purchasesAssembler from 'lib/purchases/assembler';
 import wp from 'lib/wp';
 
 const debug = debugFactory( 'calypso:upgrades:actions:purchases' ),
 	wpcom = wp.undocumented();
 
-const PURCHASES_FETCH_ERROR_MESSAGE = i18n.translate( 'There was an error retrieving purchases.' ),
-	PURCHASE_REMOVE_ERROR_MESSAGE = i18n.translate( 'There was an error removing the purchase.' );
-
 function cancelPurchase( purchaseId, onComplete ) {
```

Comment:
Well yes, but `cancelPurchase` used to call `clearPurchases` which was dispatching `PURCHASES_REMOVE`. Now that it doesn't its behavior has changed I believe. There is only `cancelPurchase` and `cancelAndRefundPurchase` left to move so we have everything from `purchases` (which was the original goal of thos PR I believe).